src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c

```diff
@@ -118,7 +119,7 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
     if (!ADUC_AgentInfo_ConnectionData_GetIntegerField(
```

Comment:
future:

`bool GetUIntField(obj, field, min, min, default, &output)`

would simply this code section, and can be used for "GetBooleanField" as well.

Reply:
tracking for future